### ISSUE TITLE
`run_look` needs limit override

### ISSUE THREAD

**Issue by githoov** [looker/lookr#11]
Regenerate Java SDK and include arguments in R wrapper function.

**Comment by maxcorbin** [comment]
Did this locally with success, will link to pull request when it is available

**Comment by maxcorbin** [comment]
This was also changed in the latest push, should be able to provide a limit override like so:

```
run_look(look_id = 1, limit = "20"
```

_Note that for now, limits have to be specified as strings. Update coming on this so users can supply a number._

**Comment by eebsidian** [comment]
Hi there,  It seems I can only retrieve 100 rows even though I specify higher values.  Is there a limitation?

**Comment by maxcorbin** [comment]
@eebsidian No, the maximum should be whatever you provide. If you run the same Look with a 100 row limit set, do you get a warning that you ht the row limit?

**Comment by Rich1000** [comment]
+1 from me

**Comment by kidman007** [comment]
+1 from me as well.

**Comment by maxcorbin** [comment]
Update on this: Due to a problem with `swagger`, the `run_look` function won't be able to support limit overrides until Looker's engineering department fleshes out the `Look` java class a bit more.

However, the limit overrides are working as expected on `run_inline_query`, so this is the workaround for now!

```
run_inline_query(model="some_model", view="some_explore", limit="100")
```

I'll be updating the repository to not accept `limit` as an argument to `run_look` until this issue can be resolved. Thanks for your patience everyone!

**Comment by Rich1000** [comment]
Any progress on this?

We've got a few people in the company that could start using R to do some really interesting stuff with our modelled data if only they could get it into R.

**Comment by githoov** [comment]
workaround: `run_look(look_id = 111, limit = "-1")`

**Comment by Rich1000** [comment]
If only I'd known! I created a very simple package to do this in the end: https://github.com/Rich1000/lookr

**Comment by mermiller** [comment]
+1 from from me too.

**Comment by Nigel-Brown** [comment]
Has any work be done to reinstate the limit option of run_look ?

**Comment by mjassows** [comment]
What we've done here as a workaround is use a little variable injection in the resultFormat string:
sdk <- lookr::LookerSDK$new(configFile = "looker.ini")
df <- sdk$runLook(lookId = 605, resultFormat = 'json?limit=99999999')
